### DISCUSSION TITLE
Prop 'rev' in icons

### DISCUSSION THREAD

**Discussion by leandoerbore** [ant-design/ant-design#44718] [Q&A] [answered]
What is the prop 'rev' in icons ? i didnt find a documentation about it.

Error:
Property 'rev' is missing in type '{}' but required in type 'Pick<AntdIconProps, "title" | "children" | "name" | "onReset" | "className" | "type" | "onClick" | "download" | "href" | "hrefLang" | "media" | "rel" | "target" | ... 350 more ... | "twoToneColor">'.<PlusOutlined /> 
```<PlusOutlined /> {title}```

**Comment by zombieJ** [comment, marked as answer]
update `@types/react`
